### PR TITLE
Updates to newest palette with updates snapshots

### PR DESCRIPTION
**Pull request by kierangillen**
- Updates snapshots from most recent palette release
- Adds ` "@artsy/palette": "4.14.7",`

### CODE REVIEW

**Comment by damassi** [comment]
👍 

merge on green

File: src/Components/v2/__tests__/CountdownTimer.test.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ describe("CountdownTimer", () => {
 
       const text = timer.text()
       expect(text).toMatchInlineSnapshot(
-        `"time remaining01d 00s leftRespond by Aug 04, 2:50 PM BSTExpired offers end the negotiation process permanently."`
```
Comment (kierangillen) [inline src/Components/v2/__tests__/CountdownTimer.test.tsx]: Reaction is still using Moment's API to mock the timezone, Palette is now using Luxon. Once we merge this PR we will update these tests.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a043e4e4db43ee707f7e5935a390b3e137bcef0c

https://deploy-preview-2509--artsy-reaction.netlify.com

**Comment by damassi** [comment]
@kierangillen - this might need to be closed and reopened as I added these components as v2 for deploys.

**Pull request closed without merging**